test/functional/rpc_fundrawtransaction.py

```diff
@@ -356,10 +354,10 @@ def run_test(self):
 
         #create same transaction over sendtoaddress
         txId = self.nodes[0].sendtoaddress(self.nodes[1].getnewaddress(), 1.1)
-        signedFee = self.nodes[0].getrawmempool(True)[txId]['fee']
+        mempool_tx = self.nodes[0].getrawmempool(True)[txId]
 
-        #compare fee
-        feeDelta = Decimal(fundedTx['fee']) - Decimal(signedFee)
+        #compare feerates
+        feeDelta = (Decimal(fundedTx['fee']) / Decimal(len(fundedTx['hex']) / 2)) - (Decimal(mempool_tx['fee']) / mempool_tx['size'])
```

Comment:
Shouldn't this be segwit sensitive? Hexlength is only a valid indicator of weight for non-segwit transactions.

Reply:
The test only uses legacy addresses. I don't think it is necessary to make a helper function for this.